### PR TITLE
Mod

### PR DESCRIPTION
**Pull request by gv2011**

### CODE REVIEW

**Comment by gv2011** [comment]
lala

**Comment by oleg-nenashev** [comment]
Sorry, but it cannot be accepted in the current state 
The PR needs a description and a better justification of pom.xml changes

File: core/src/test/java/hudson/util/XStream2Test.java [hunk 1]
```diff
@@ -23,24 +23,29 @@
  */
 package hudson.util;
 
-import static org.junit.Assert.*;
-
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import com.thoughtworks.xstream.XStreamException;
+import static org.junit.Assert.assertEquals;
```
Comment (KostyaSha) [inline core/src/test/java/hudson/util/XStream2Test.java]: No need in replacing static import with `*`

Comment (gv2011) [inline core/src/test/java/hudson/util/XStream2Test.java]: Sorry, I made this pull request accidently.

Am 01.07.15 KW26 um 20:33 schrieb Kanstantsin Shautsou:

**Pull request closed without merging**